.gitignore

```diff
@@ -2,3 +2,4 @@
 *.pyc
 .DS_Store
 _site
+.idea/
```

Comment:
Pushed commit removing .DS_Store (which is platform/machine-specific).

Didn't cut .pyc in this commit (those files would reasonably be created by normal use of this repo, across any platform).

Hope that's a reasonable threshold? Didn't mean to sidetrack review of this commit.